doc/api/cli.md

```diff
@@ -1535,6 +1535,14 @@ environment variables.
 
 See `SSL_CERT_DIR` and `SSL_CERT_FILE`.
 
+### `--node-use-system-ca`
+
+Node.js uses the trusted CA certificates present in the system store along with
+the `--use-bundled-ca`, `--use-openssl-ca` options.
+
+Only current user certificates are accessible using this method, not the local
+machine store. This option is available to Windows only.
```

Comment:
```suggestion
Only current user certificates are accessible using this method, not the local
machine store.

This option is available to Windows only.
```

It's worth separating out the "Windows only" line so it's more visible.

Reply:
Yeah agreed. It's done